spec/features/admin_users_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'capybara_helper'
+
+describe Admin::UsersController do
```

Comment:
We should also test that non-admins cannot access this, or add that in a controller spec.